src/visualization/utils/getFormatter.ts

```diff
@@ -30,6 +30,12 @@ export const getFormatter = (
     format,
   }: GetFormatterOptions = {}
 ): null | ((x: any) => string) => {
+  // timeFormat was being passed in an empty string, which meant TypeScript will not
+  // replace it with the default, so we need to do it here with a check.
+  if (timeFormat === '') {
```

Comment:
I realize I am modifying the passed-in function parameter, it seemed like a better idea than creating a new variable and naming it something else. But I can change the approach if this is not acceptable.